gradio/components/chatbot.py

```diff
@@ -184,18 +209,36 @@ def preprocess(
         return processed_messages
 
     def _postprocess_chat_messages(
-        self, chat_message: str | tuple | list | None
-    ) -> str | FileMessage | None:
-        if chat_message is None:
-            return None
-        elif isinstance(chat_message, (tuple, list)):
-            filepath = str(chat_message[0])
-
+        self, chat_message: str | tuple | list | GradioComponent | None
+    ) -> str | FileMessage | List[FileMessage] | PlotData | GalleryData | None:
+        def create_file_message(chat_message, filepath):
             mime_type = client_utils.get_mimetype(filepath)
             return FileMessage(
                 file=FileData(path=filepath, mime_type=mime_type),
-                alt_text=chat_message[1] if len(chat_message) > 1 else None,
+                alt_text=chat_message[1]
```

Comment:
Imo we should not support passing a gradio component in a tuple. Makes it so that there are too many ways to do the same thing. I think just supporting the value outside the tuple is enough.